traffic-dashboard/src/components/navbar/navbar.tsx

```diff
@@ -78,6 +79,14 @@ export const Navbar: React.FC = () => {
                 toggleColorMode={toggleColorMode}
                 isDarkMode={mode === "dark"}
               />
+              <a
+                href="https://discord.gg/78KY7GW2WW"
+                className={styles.discord_icon}
+              >
+                <SvgIcon>
```

Comment:
would love it if you could move this svg to another file. maybe an assets folder.